Articles/Packages/Extending/index.md

```diff
@@ -0,0 +1,80 @@
+# Packages
+
+## What is a Package?
+
+A package extends Umbraco to provide additional functionality to editors, developers, site visitors, and all other types of users of Umbraco. It can impact one or more of these groups of people depending on the type of package.
+
+An Umbraco Package can be many things, but are generally characterized by:
+
+- Adding or extending functionality in the Umbraco CMS
+- Empowering people to do more and/or do things more efficiently
+- Engaging community members in collaboration and sharing
+- Solving real life problems
+- Inspiring people on what Umbraco can be made capable of
```

Comment:
```suggestion
- Inspiring people on what Umbraco can be capable of
```

Reply:
Isn't what we're trying to say that, through package developers help Umbraco can be made capable of amazing things? Rather than Umbraco just being inherently awesome? (Which it is)

Reply:
Agreed, I'll remove this suggestion.